src/main/scala/is/hail/HailContext.scala

```diff
@@ -584,12 +585,14 @@ class HailContext private(val sc: SparkContext,
     missingVal: String = "NA",
     nPartitions: Option[Int],
     noHeader: Boolean,
-    forceBGZ: Boolean): MatrixTable = {
+    forceBGZ: Boolean,
+    sep: String = "\t"): MatrixTable = {
+    assert(sep.length == 1)
 
     val inputs = hadoopConf.globAll(files)
 
     forceBGZip(forceBGZ) {
-      LoadMatrix(this, inputs, rowFields, keyNames, cellType = TStruct("x" -> cellType), missingVal, nPartitions, noHeader)
+      LoadMatrix(this, inputs, rowFields, keyNames, cellType = TStruct("x" -> cellType), missingVal, nPartitions, noHeader, sep(0))
```

Comment:
can you enforce here that `sep` is only one character long?

Reply:
I enforce in python here: https://github.com/hail-is/hail/pull/3444/files#r184263112

Reply:
oops. thanks!